client/my-sites/upgrades/domain-management/email/google-apps-users-card.jsx

```diff
@@ -27,11 +28,14 @@ const GoogleAppsUsers = React.createClass( {
 		user: React.PropTypes.object.isRequired
 	},
 
-	canAddUsers() {
-		const domainsInContext = this.props.selectedDomainName
+	getDomains() {
```

Comment:
It's not clear what this function does on the first look. It's named `getDomains`, uses props `selectedDomainName` and `domains` and a `getSelectedDomain` function.

Although I know what it does since I've been playing around with this, adding a bit of comment would be helpful for others.